src/client/pages/Offer/Checkout/index.tsx

```diff
@@ -236,13 +236,6 @@ export const Checkout = ({
   const isDiscountMonthlyCostDeduction =
     campaign?.incentive?.__typename === 'MonthlyCostDeduction'
 
-  const offerData = useMemo(
```

Comment:
I needed to remove this as we're calculating `offerData` info twice (at `Offer` and `Checkout` levels) and while changing star date were triggering an update of `offerData` on `Offer` level the same wasn't happening at `Checkout` level due the memoization, which were resulting in a inconsistent sate between the date pickers present on `Sidebar` and `Checkout` components.

Reply:
Hmmm I guess our assumptions were wrong :P